changelog/0075-reindex-all-indexes-for-es-upgrade.yml

```diff
@@ -0,0 +1,85 @@
+title: reindex-all-indexes-for-es-upgrade
+key: reindex-all-indexes-for-es-upgrade
+date: 2023-10-25
+optional_per_env: no
+# (optional) Min version of HQ that MUST be deployed before this change can be rolled out (commit hash)
+min_commcare_version:
+# I will add it after the PR is merged
+max_commcare_version:
+context: |
+  Reindex Elasticsearch Indices for upcoming ES Upgrade.
+details: |
+  Currently Commcare HQ is running with Elasticsearch 2.4. Reindexing is required before upgrading to a newer version of Elasticsearch 5.6.16
+update_steps: |
+  1. Deploy latest version of Commcare HQ.
+  2. Ensure that there is enough free space available in your cluster. The command to estimate disk space required for reindexing is:
+
+  ```sh
+  cchq <env> django-manage elastic_sync_multiplexed estimated_size_for_reindex
+  ```
+  3. Check disk usage on each node
+    - If you have sepearate data nodes, check disk usage on data nodes
+      ```sh
+      cchq <env>  run-shell-command es_data "df -h /opt/data" -b
+      ```
+    - If you don't have separate data nodes, check disk usage on ES nodes.
+      ```sh
+      cchq <env>  run-shell-command es_data "df -h /opt/data" -b
+      ```
```

Comment:
The commands in these two points are identical. If that was expected, would it be possible to combine the two points into one?

Reply:
Nope. It should have been 
```sh
      cchq <env>  run-shell-command elasticsearch "df -h /opt/data" -b
```

Reply:
Done in https://github.com/dimagi/commcare-cloud/pull/6151/commits/65ed26f92271a8d9e0e5720481f3824220783059#diff-0b554b508b3d07a7ab1f2530b4d2c79ab58443d342938372c12a0aedfa015863R26